public/src/components/channelManagement/bannerDesigns/HeaderImageEditor.tsx

```diff
@@ -0,0 +1,104 @@
+import { TextField } from '@material-ui/core';
+import { EMPTY_ERROR_HELPER_TEXT } from '../helpers/validation';
+import React, { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { BannerDesignImage } from '../../../models/bannerDesign';
+
+const imageUrlValidation = {
+  value: /^https:\/\/i\.guim\.co\.uk\//,
+  message: 'Images must be valid URLs hosted on https://i.guim.co.uk/',
+};
```

Comment:
Not a big deal, but perhaps this common validation could be shared with the other image fields?